src/redux/reducers/index.js

```diff
@@ -0,0 +1,168 @@
+const initialState = {
+  products: [],
+  cart: [],
+  totalAmount: 0,
+  filteredProducts: [],
+  filters: {
+    category: [],
+    priceUnder: [],
+    sortOrder: null,
+  },
+  address: null,
+  theme: "light",
+};
+
+const reducer = (state = initialState, action) => {
+  switch (action.type) {
+    case "ADD_TO_CART": {
```

Comment:
@arun-121 can you move these to constants so that the same values can be used in actions and in reducers

Reply:
moved constants to a separate folder